api/google_analytics_provider.py

```diff
@@ -13,11 +13,22 @@
 )
 
 class GoogleAnalyticsProvider(object):
-    NAME = _("Google Analytics")
 
-    DESCRIPTION = _("<p>Send circulation-related analytics events to Google Analytics." +
-                    "<p>To use this integration, set up a property in Google Analytics. " +
-                    "On the administration page for the property, go to Custom Definitions > Custom Dimensions, " +
+    NAME = _("Google Analytics")
+    DESCRIPTION = _("How to Configure a Google Analytics Integration")
+    INSTRUCTIONS = _("<p>In order to track usage statistics, you can configure the Circulation Manager " +
+                    "to connect to Google Analytics.  Currently, Google Analytics is the only analytics engine " +
+                    "supported for this function.</p>" +
+                    "<p>There are two components required to use Google Analytics to capture usage from your SimplyE service: " +
+                    "<ol><li><a href='https://analytics.google.com/analytics/web/provision/?authuser=0#/provision/create' " +
+                    "rel='noopener' rel='noreferer' target='_blank'>Create a Google Analytics account</a> and configure it " +
+                    "to accommodate elements of SimplyE usage</li>" +
```

Comment:
"SimplyE" should be changed to "Library Simplified" throughout this since many of the events don't come from the app.